web-marketplace/src/components/organisms/ChooseCreditsForm/ChooseCreditsForm.tsx

```diff
@@ -249,7 +249,7 @@ export function ChooseCreditsForm({
         </Card>
         <div className="float-right pt-40">
           <PrevNextButtons
-            saveDisabled={!isValid || isSubmitting}
+            saveDisabled={Object.entries(errors).length > 0 || isSubmitting}
```

Comment:
not entirely sure why isValid is not synced with errors here 
this seems to be an on going issue: https://github.com/react-hook-form/react-hook-form/issues/2755
but this fixes the issue, let's just be careful this doesn't have any side effect